README.md

```diff
@@ -14,56 +14,73 @@ Small cli toolbox for creating cross-browser WebExtensions.
 If you want to get started quickly check out the [yeoman generator](https://github.com/webextension-toolbox/generator-web-extension) for this project.
 
 # Browser Support
-* `chrome` (auto [polyfilled](https://github.com/mozilla/webextension-polyfill))
-* `opera` (auto [polyfilled](https://github.com/mozilla/webextension-polyfill))
-* `firefox`
-* `edge`
+
+* `Chrome` (auto [polyfilled](https://github.com/mozilla/webextension-polyfill))
+* `Opera` (auto [polyfilled](https://github.com/mozilla/webextension-polyfill))
+* `Firefox`
+* `Edge`
```

Comment:
Hi @dotproto,

Could you please convert these back to lowercase, as these are used lowercase in the tool as well?
Or it would be even better to have like this: Chrome `chrome`